playlists/excel.yaml

```diff
@@ -9,6 +9,7 @@
   group: Basics
   api_set:
     ExcelApi: 1.1
+  type: public
```

Comment:
Since this playlist is only going to have public snippets in it, maybe get rid of the `type` property before dumping to yaml?  Just for cleanliness sake...